docs/UsersGuide/source/BuildingRunningTesting/ContainerQuickstart.rst

```diff
@@ -76,23 +74,26 @@ Level 1 Systems
 
 On most Level 1 systems, a container named ``ubuntu20.04-intel-ue-1.4.1-srw-dev.img`` has already been built at the following locations:
 
-.. table:: Locations of pre-built containers
-
-   +--------------+--------------------------------------------------------+
-   | Machine      | File location                                          |
-   +==============+========================================================+
-   | Cheyenne     | /glade/scratch/epicufsrt/containers                    |
-   +--------------+--------------------------------------------------------+
-   | Gaea         | /lustre/f2/dev/role.epic/containers                    |
-   +--------------+--------------------------------------------------------+
-   | Hera         | /scratch1/NCEPDEV/nems/role.epic/containers            |
-   +--------------+--------------------------------------------------------+
-   | Jet          | /mnt/lfs4/HFIP/hfv3gfs/role.epic/containers            |
-   +--------------+--------------------------------------------------------+
-   | NOAA Cloud   | /contrib/EPIC/containers                               |
-   +--------------+--------------------------------------------------------+
-   | Orion        | /work/noaa/epic-ps/role-epic-ps/containers             |
-   +--------------+--------------------------------------------------------+
+.. list-table:: Locations of pre-built containers
+   :widths: 20 50
+   :header-rows: 1
+
+   * - Machine
+     - File Location
+   * - Cheyenne/Derecho [#fn]_
+     - /glade/scratch/epicufsrt/containers
```

Comment:
@gspetro-NOAA -

With Cheyenne decommissioned, should we change the container path to `/glade/work/epicufsrt/contrib/containers`?